percentile.go

```diff
@@ -18,6 +18,10 @@ func Percentile(input Float64Data, percent float64) (percentile float64, err err
 	// Start by sorting a copy of the slice
 	c := sortedCopy(input)
 
+	if len(c) == 1 {
```

Comment:
It's really micro optimization but maybe better move this check before sorting?

Reply:
ok

Reply:
> It's really micro optimization but maybe better move this check before sorting?

done